p2p-specs/p2p-interface.md

```diff
@@ -165,26 +167,40 @@ For any optional queueing, Bundlers SHOULD maintain maximum queue sizes to avoid
 
 #### Global topics
 
-There are two primary global topics used to propagate user operations (`UserOperationWithEntryPoint`)
-and aggregate user operation hashes (`NewPooledUserOperationsHashes`) to all nodes on the network.
+The primary global topics used to propagate user operations to all nodes on the network is `UserOperationWithEntryPoint`.
 
 ##### `user_ops_with_entry_point`
 
 The `user_ops_with_entry_point` topic is the concatenation of EntryPoint address and UserOperation message serialized using SSZ
 
 The following validations MUST pass before forwarding the `user_ops_with_entry_point` on the network
-- TBD
-- TBD
-- TBD
+- [IGNORE] `verified_at_block_hash` is too far in the past.
+- [REJECT] If any of the sanity checks specified in the [EIP](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4337.md#client-behavior-upon-receiving-a-useroperation) fails.
+- [REJECT] If the simulated user operation fails.
+
+### Encodings
+
+Topics are post-fixed with an encoding. Encodings define how the payload of a gossipsub message is encoded.
+
+ssz_snappy - All objects are SSZ-encoded and then compressed with Snappy block compression. Example: The user_ops_with_entry_point topic string of the canonical mempool is /account_abstraction/<mempool_id>/user_ops_with_entry_point/ssz_snappy, the <mempool_id> is <> and the data field of a gossipsub message is an UserOpsWithEntryPoiint that has been SSZ-encoded and then compressed with Snappy.
```

Comment:
`the <mempool_id> is <>`

Why `<>`?  The canonical mempool id of each chain should be different, since they obviously shouldn't be mixed.  The description of the mempool id contains the chain id, so it'll be different in any case.

Reply:
> Why `<>`? The canonical mempool id of each chain should be different, since they obviously shouldn't be mixed. The description of the mempool id contains the chain id, so it'll be different in any case.

I was hoping to replace `<>` with the actual mempool-id corresponding to the canonical mempool of Ethereum mainnet. For this we need to finalise/free the JSON structure of the mempool file. Once that is done, I'll add the file to IPFS and provide the IPFS hash here.